app/models/message.rb

```diff
@@ -21,12 +21,12 @@ class Message < ActiveRecord::Base
   extend Hideable::ActiveRecord
   hideable
 
-  belongs_to :recipient, :class_name => "User"
-  belongs_to :submitter, :class_name => "User"
+  belongs_to :recipient, :class_name => User
+  belongs_to :submitter, :class_name => User
```

Comment:
[pronto-rubocop warning] Rails/ReflectionClassName: Use a string value for `class_name`.